client/src/components/RequestManagementContainer.tsx

```diff
@@ -0,0 +1,271 @@
+import React, {Component} from 'react';
+import {Grid} from "semantic-ui-react";
+import RequestManagementBoard from "./RequestManagement/RequestManagementBoard";
+import {RequestedItem, ItemStatus} from "./inventory/HardwareItem"
+
+const boardStyle = {
```

Comment:
use CSS for these styles (make a new .scss file in the `styles` directory and then `@include` the new file in `index.scss`)